sell-loot.lic

```diff
@@ -82,6 +98,9 @@ class SellLoot
     return unless @bankbot_room_id
     return unless @bankbot_deposit_threshold
 
+    tip_currency = @town_data[hometown]['currency']
+    keep_copper = @bankbot_deposit_threshold*-1 if keep_copper == -1
+
```

Comment:
The only thing we care about hometown for in this method was pulling out currency and copper on hand. It seems like it makes more sense to rename this method to `give_money_to_bankbot(currency, keep)` and kill the keep_copper hack and kill the `@bankbot_deposit_threshold` guard at the top of this method.

Doing that we change the above calls to the more reasonable 
```
['Lirums', 'Kronars', 'Dokoras'].each{|currency| give_money_to_bankbot(currency, currency == real_currency ? @bankbot_deposit_threshold : 0) }`